Formula/g/gerbil-scheme.rb

```diff
@@ -27,37 +27,47 @@ class GerbilScheme < Formula
     sha256 x86_64_linux:   "878b862448fe401b00980688c6c880ef4344cc88272bb29ed6c1ddb1ce14460f"
   end
 
-  depends_on "gambit-scheme"
-  depends_on "leveldb"
-  depends_on "libyaml"
-  depends_on "lmdb"
+  depends_on "coreutils" => :build
+  depends_on "pkgconf" => :build
+
   depends_on "openssl@3"
 
-  uses_from_macos "libxml2"
   uses_from_macos "sqlite"
   uses_from_macos "zlib"
 
   on_macos do
     depends_on "gcc"
   end
 
+  # TODO: add `conflicts_with "ghostscript"`
+  conflicts_with "gambit-scheme", because: "both install `gsc` binary"
+
   fails_with :clang do
     cause "gambit-scheme is built with GCC"
   end
 
   def install
-    cd "src" do
-      system "./configure", "--prefix=#{prefix}",
-                            "--with-gambit=#{Formula["gambit-scheme"].opt_prefix}",
-                            "--enable-leveldb",
-                            "--enable-libxml",
-                            "--enable-libyaml",
-                            "--enable-lmdb"
-      system "./build.sh"
-      system "./install"
+    system "./configure", "--prefix=#{prefix}", "--enable-march="
+    ENV.deparallelize
+    system "make"
+    system "make", "install"
+
+    # `make install` command creates a directory `v#{version}` with `bin`, `include`, `lib`, and other directories
+    # then in creates symlinks in root prefix directory
 
-      mv "#{share}/emacs/site-lisp/gerbil", "#{share}/emacs/site-lisp/gerbil-scheme"
+    # 1. Remove all symlinks
+    %w[bin current include lib share src].each do |symlink|
+      rm prefix/symlink
     end
+
+    # 2. Install files manually
+    bin.install Dir.glob("#{prefix}/v#{version}/bin/*")
+    include.install Dir.glob("#{prefix}/v#{version}/include/*")
+    elisp.install Dir.glob("#{prefix}/v#{version}/share/emacs/site-lisp/*")
+
+    # Install libraries as symlink because binaries are already linked to
+    # $HOMEBREW_PREFIX/Cellar/gerbil-scheme/<version>/v<version>/lib/<lib>
+    lib.install_symlink Dir.glob("#{prefix}/v#{version}/lib/*")
```

Comment:
```suggestion
    bin.install (prefix/"v#{version}/bin").children
    include.install (prefix/"v#{version}/include").children
    elisp.install (prefix/"v#{version}/share/emacs/site-lisp").children

    # Install libraries as symlink because binaries are already linked to
    # $HOMEBREW_PREFIX/Cellar/gerbil-scheme/<version>/v<version>/lib/<lib>
    lib.install_symlink (prefix/"v#{version}/lib").children
```